src/utils.ts

```diff
@@ -1847,39 +1856,36 @@ export class newText extends newShape {
   }
 }
 
-export interface pointStyle {
-  size?: number, fillStyle?: string, strokeStyle?: string, marker?: string, markerOrientation?: string
+export interface PointStyleInterface {
+  size?: number,
+  color_fill?: string,
+  color_stroke?: string,
+  stroke_width?: number,
+  shape?: string,
+  name?: string
 }
 
-export class newPointStyle implements pointStyle {
+export class newPointStyle implements PointStyleInterface {
   public size: number;
   public fillStyle: string;
   public strokeStyle: string;
   public marker: string;
-  public markerOrientation: string;
+  public lineWidth: number;
   constructor(
     { size = null,
-      fillStyle = null,
-      strokeStyle = null,
-      marker = 'circle',
-      markerOrientation = 'up'
-    }: pointStyle = {}
+      color_fill = null,
+      color_stroke = null,
+      stroke_width = null,
+      shape = 'circle',
+      name = '',
+    }: PointStyleInterface = {}
     ) {
       this.size = size;
-      this.fillStyle = fillStyle;
-      this.strokeStyle = strokeStyle;
-      this.marker = marker;
-      this.markerOrientation = markerOrientation;
+      this.fillStyle = color_fill;
+      this.strokeStyle = color_stroke;
+      this.marker = shape;
+      this.lineWidth = stroke_width;
```

Comment:
This will not work when creating PointStyle from Typescript interface and will change when this need will appear